crates/oxc_linter/src/rules/eslint/no_unused_vars/usage.rs

```diff
@@ -818,4 +841,20 @@ impl<'a> Symbol<'_, 'a> {
             };
         }
     }
+
+    /// Return the **variable scope** for the given `scope_id`.
+    ///
+    /// A variable scope is the closest ancestor scope (including `scope_id`
+    /// itself) whose kind can *outlive* the current execution slice:
+    ///   * function‑like scopes  
+    ///   * class static blocks  
+    ///   * TypeScript namespace/module blocks
+    fn get_parent_variable_scope(&self, scope_id: ScopeId) -> ScopeId {
+        self.scoping()
+            .scope_ancestors(scope_id)
+            .find_or_last(|scope_id| {
+                self.scoping().scope_flags(*scope_id).intersects(ScopeFlags::Var)
```

Comment:
```suggestion
                self.scoping().scope_flags(*scope_id).is_var()
```

Sorry, I am forgot to mention you can use `ScopeFlags::is_var` directly.